api/README.md

```diff
@@ -0,0 +1,114 @@
+# api
+// TODO(user): Add simple overview of use/purpose
+
+## Description
+// TODO(user): An in-depth paragraph about your project and overview of use
+
+## Getting Started
+
+### Prerequisites
+- go version v1.22.0+
+- docker version 17.03+.
+- kubectl version v1.11.3+.
+- Access to a Kubernetes v1.11.3+ cluster.
+
+### To Deploy on the cluster
```

Comment:
can we update this with the instructions to deploy the CRDs?

Reply:
Added to the top-level README